### ISSUE TITLE
インストールエラー：ENOENT: no such file or directory, open './package-lock.json'

### ISSUE THREAD

**Issue by PokeyTheCat** [ukyoda/ogpParser#28]
```
PS C:\Users\user> npm i -global ogp-parser

> ogp-parser@0.5.1 preinstall C:\Users\user\AppData\Roaming\npm\node_modules\ogp-parser
> npx npm-force-resolutions

npx: installed 5 in 1.729s
ENOENT: no such file or directory, open './package-lock.json'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ogp-parser@0.5.1 preinstall: `npx npm-force-resolutions`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ogp-parser@0.5.1 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Roaming\npm-cache\_logs\2020-07-21T02_58_56_906Z-debug.log
```

githubからrepositoryをダウンロードしてとlocalインストーしました：

```
PS C:\Users\user> cd C:\Users\user\Downloads\ogpParser-master
PS C:\Users\user\Downloads\ogpParser-master> npm install -g

> ogp-parser@0.5.1 preinstall C:\Users\user\AppData\Roaming\npm\node_modules\ogp-parser
> npx npm-force-resolutions

npx: installed 5 in 1.813s

> fast-xml-parser@3.17.4 postinstall C:\Users\user\Downloads\ogpParser-master\node_modules\fast-xml-parser
> node tasks/postinstall.js || exit 0

Love fast-xml-parser? Check https://amitkumargupta.work for more projects and contribution.

+ ogp-parser@0.5.1
added 29 packages from 67 contributors in 4.193s
PS C:\Users\user\Downloads\ogpParser-master>

```
それはいいけどdependenciesが無いです。
```
PS C:\Users\user\AppData\Roaming\npm> node.exe .\test.js
internal/modules/cjs/loader.js:1033
  throw err;
  ^

Error: Cannot find module 'he'
Require stack:
- C:\Users\user\Downloads\ogpParser-master\ogpParser.js
- C:\Users\user\AppData\Roaming\npm\test.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1030:15)
    at Function.Module._load (internal/modules/cjs/loader.js:899:27)
    at Module.require (internal/modules/cjs/loader.js:1090:19)
    at require (internal/modules/cjs/helpers.js:75:18)
    at Object.<anonymous> (C:\Users\user\Downloads\ogpParser-master\ogpParser.js:9:12)
    at Module._compile (internal/modules/cjs/loader.js:1201:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1221:10)
    at Module.load (internal/modules/cjs/loader.js:1050:32)
    at Function.Module._load (internal/modules/cjs/loader.js:938:14)
    at Module.require (internal/modules/cjs/loader.js:1090:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\user\\Downloads\\ogpParser-master\\ogpParser.js',
    'C:\\Users\\user\\AppData\\Roaming\\npm\\test.js'
  ]
}
```
手伝ってお願いします。ORZ
英語はいいですか？

**Comment by PokeyTheCat** [comment]
So I've gotten my copy from the repo working by installing the `he` module into `C:\Users\user\Downloads\ogpParser-master\node_modules` instead of my global modules located at `C:\Users\user\AppData\Roaming\npm\node_modules`.

Maybe something is wrong with the way I initially installed node and npm. I will see if I get the same results on my linux workstation with `npm i ogp-parser`.

**Comment by ukyoda** [comment]
I have released new version `v0.5.2`, which adds the `he` to the dependencies.

Please You tried the new version `v0.5.2`

Thank you!!

**Comment by PokeyTheCat** [comment]
`v0.5.2` fixes the `he` dependency when I install from your git repo.

However if I try to use the package from npm instead of cloning the git repository I get this error:

```
PokeyTheCat@gollum:~/nodeTest$ npm i ogp-parser

> ogp-parser@0.5.2 preinstall /home/PokeyTheCat/nodeTest/node_modules/ogp-parser
> npx npm-force-resolutions

ENOENT: no such file or directory, open './package-lock.json'
npm WARN nodeTest@1.0.0 No description
npm WARN nodeTest@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ogp-parser@0.5.2 preinstall: `npx npm-force-resolutions`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ogp-parser@0.5.2 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/PokeyTheCat/.npm/_logs/2020-07-25T05_16_16_053Z-debug.log
```
As you can tell I'm not an experienced node developer, but perhaps `package-lock.json` should be published when you push your build to the npm repo?

I used `npm view ogp-parser` and downloaded from `https://registry.npmjs.org/ogp-parser/-/ogp-parser-0.5.2.tgz`. I extracted the tarball and copied the package-lock.json from the git repository and created a test tarball and installed from that.

```
PokeyTheCat@gollum:~/nodebody$ cp ~/ogpParser/package-lock.json package/
PokeyTheCat@gollum:~/nodebody$ tar -cvzf ogp-parser-test.tgz package
package/
package/package.json
package/README.md
package/example/
package/example/app.js
package/example/app-old.js
package/old/
package/old/ogpParser.js
package/test/
package/test/old/
package/test/old/request.js
package/test/data/
package/test/data/euc_jp.txt
package/test/utils.js
package/test/html/
package/test/html/demo_oembed.html
package/test/html/demo.html
package/test/html/oembed.json
package/test/html/oembed.xml
package/test/html/demo_oembed_xml.html
package/test/request.js
package/test/parseHTML.js
package/package-lock.json
package/utils/
package/utils/oembed.js
package/utils/charsetConverter.js
package/utils/getContents.js
package/utils/parseHtml.js
package/.editorconfig
package/.jshintrc
package/docs/
package/docs/old-version.md
package/LICENSE
package/ogpParser.js
PokeyTheCat@gollum:~/nodebody$ ls
node_modules  npm-debug.log  ogp-parser-0.5.2.tgz  ogp-parser-test.tgz  package  package.json  package-lock.json  test.js
PokeyTheCat@gollum:~/nodebody$ npm i ogp-parser-test.tgz

> ogp-parser@0.5.2 preinstall /home/PokeyTheCat/nodebody/node_modules/ogp-parser
> npx npm-force-resolutions


> fast-xml-parser@3.17.4 postinstall /home/PokeyTheCat/nodebody/node_modules/fast-xml-parser
> node tasks/postinstall.js || exit 0

Love fast-xml-parser? Check https://amitkumargupta.work for more projects and contribution.

npm WARN nodebody@1.0.0 No description
npm WARN nodebody@1.0.0 No repository field.

+ ogp-parser@0.5.2
added 30 packages from 67 contributors and audited 35 packages in 5.322s

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

```

**Comment by ukyoda** [comment]
@PokeyTheCat 

Thanks!!

The problem was caused by the `preinstall` command `npx npm-force-resolutions` in the package.json scripts.

I have removed the `preinstall` command, and I have released the new version `v0.5.3`.

I have confirmed that this version can be `npm installed` with no problems in my development environment (Mac, node version=v13.0 and npm version=6.14.7)

Please You tried the new version v0.5.3

**Comment by PokeyTheCat** [comment]
`v0.5.3` works for me in both of my environments.

お疲れ様でした！
